.golangci.yml

```diff
@@ -0,0 +1,17 @@
+issues:
+        max-same-issues: 0
+        exclude-use-default: false
+linters:
+        enable:
+        - whitespace
+        - gosec
+        - gci
+        - misspell
+        - gomnd
+        - gofmt
+        - goimports
```

Comment:
I don't think this is actually helpful, because both missing and unused imports are already a build error. AFAIK goimports is useful while developing code, to fix those missing/unused build errors. It doesn't really help for CI as those cases will already fail.